src/sentry/statistical_detectors/detector.py

```diff
@@ -70,80 +75,8 @@ def from_dict(d: Any) -> TrendState:
 
 @dataclass
 class TrendPayload:
+    project_id: int
```

Comment:
if the `project_id` is in the key necessary to retrieve the trend payload, why does it need to be in the value as well?